components/NavBarButtonVisita.tsx

```diff
@@ -28,7 +28,9 @@ const NavBarButtonVisita = () => {
           height: "80px",
         }}
       >
-        <AccessTimeIcon sx={{ color: "#fff", float: "left", mt: -0.5,ml:3 }} />
+        <AccessTimeIcon
```

Comment:
Negative margins are not the best solution. But let this way and we'll go back on that later.